homeassistant/components/volvooncall.py

```diff
@@ -123,6 +126,13 @@ def __init__(self, hass, vin, attribute):
         self._state.entities[self._vin].append(self)
 
     @property
+    def entity_id(self):
```

Comment:
Please help me understand why you think it is wrong to keep it in the VolvoEntity base class (base class for both the implementation within sensor/ and binary_sensor/)?